libcloud/compute/drivers/outscale.py

```diff
@@ -412,23 +457,80 @@ def create_node(self,
             data.update({"SubnetId": ex_subnet_id})
         action = "CreateVms"
         data = json.dumps(data)
-        return self._to_node(self._call_api(action, data).json()["Vms"][0])
+        node = self._to_node(self._call_api(action, data).json()["Vms"][0])
+        if name is not None:
+            action = "CreateTags"
+            data = {
+                "DryRun": ex_dry_run,
+                "ResourceIds": [node.id],
+                "Tags": {
+                    "Key": "Name",
+                    "Value": name
+                }
+            }
+            data = json.dumps(data)
+            if self._call_api(action, data).status_code != 200:
```

Comment:
To make troubleshooting easier, it would probably be better to throw an exception with the underlying error message /   response.